youtube_bz/main.py

```diff
@@ -20,6 +20,7 @@ async def run_command(args: argparse.Namespace):
         await commands.download(
             args.mbid,
             verbose,
+            destination=args.destination
```

Comment:
`args.destination` is explicit enough i don't think we need to use a named parameters here.